src/app/blog/iroh-0-21-fix-clean-polish/page.mdx

```diff
@@ -0,0 +1,128 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import { MotionCanvas } from '@/components/MotionCanvas'
+
+export const post = {
+  draft: false,
+  author: 'dig',
+  date: '2024-07-22',
+  title: 'iroh 0.21.0 - Fix, clean & polish',
+  description: 'Iroh 0.21.0 release',
+}
+
+export const metadata = {
+    title: post.title,
+    description: post.description,
+    openGraph: {
+      title: post.title,
+      description: post.description,
+      images: [{
+        url: `/api/og?title=Blog&subtitle=${post.title}`,
+        width: 1200,
+        height: 630,
+        alt: post.title,
+        type: 'image/png',
+      }],
+      type: 'article'
+    }
+}
+
+export default (props) => <BlogPostLayout article={post} {...props} />
+
+Welcome to a new release of iroh, the open-source distributed systems toolkit with tools for direct connections, moving data, and syncing state.
+
+This time around a lot of work went into fixing subtle bugs, improving documentation, and polishing APIs. We hope this will help you to enjoy iroh even more!
```

Comment:
```suggestion
This time around a lot of work went into fixing subtle bugs, improving documentation, and polishing APIs. We hope this will help you enjoy iroh even more!
```